cypress/integration/regression/decision-tree-regression.spec.ts

```diff
@@ -0,0 +1,488 @@
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.decisionTreeResults.leftInput.type('10')
+    decisionTreesPage.decisionTreeResults.rightInput.type('100')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    decisionTreesPage.decisionTreeResults.searchGraphResults
+      .eq(1)
+      .type('Min_GQ')
+    decisionTreesPage.decisionTreeResults.stepCard
+      .getChildren()
+      .contains('Min_GQ')
+    decisionTreesPage.decisionTreeResults.searchGraphResults.eq(1).clear()
+    decisionTreesPage.decisionTreeResults.addAttribute.eq(1).click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('Max_GQ')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.decisionTreeResults.leftInput.type('15')
+    decisionTreesPage.decisionTreeResults.rightInput.type('99')
+    decisionTreesPage.attributesList.replaceButton.click()
+    decisionTreesPage.decisionTreeResults.searchGraphResults
+      .eq(1)
+      .type('Max_GQ')
+    decisionTreesPage.decisionTreeResults.stepCard
+      .getChildren()
+      .contains('Max_GQ')
+  })
+
+  it('should add third step | step 9', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains(selectAll).click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeResults.excludeInfo
+      .first()
+      .should('have.text', includedVariants)
+    cy.intercept('POST', '/app/dtree_stat').as('stepAfter')
+    decisionTreesPage.decisionTreeResults.optionsMenu.click()
+    decisionTreesPage.decisionTreeResults.addStepAfter.click()
+    cy.wait('@stepAfter')
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(2)
+    decisionTreesPage.decisionTreeResults.addAttribute.eq(1).click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('Min_GQ')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.decisionTreeResults.leftInput.type('10')
+    decisionTreesPage.decisionTreeResults.rightInput.type('100')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@stepAfter')
+    decisionTreesPage.decisionTreeResults.optionsMenu.eq(1).click()
+    decisionTreesPage.decisionTreeResults.addStepAfter.click()
+    cy.wait('@stepAfter')
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(3)
+  })
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('should add attributes to step 3 | step 10', () => {
```

Comment:
Why it's disabled?

Reply:
there is a bug in an interface or a plug instead of functionality which will be implemented in nearest future